docs/.eleventy.js

```diff
@@ -34,7 +34,7 @@ module.exports = function(eleventyConfig) {
 
     let pathPrefix = "/docs/head/";
 
-    if (process.env.CONTEXT === "deploy-preview") {
+    if (process.env.CONTEXT === "deploy-preview" || process.env.BRANCH === "main") {
         pathPrefix = "/";
```

Comment:
I think this would break all pages under https://eslint.org/docs/head/

Reply:
You're right. The change is wrong, I missed something.

I took a closer look, and now I have a problem. zh-hans site uses the exact same code as this, but they both have different resource prefixes.

The root site's resources are under `/docs/head/`, while the zh-hans site's resources are under `/`. Both of them are `HEAD` version. Is there a detail I'm missing?

Reply:
Maybe we could use an environment variable that would override this logic?